wiggum_app/models.py

```diff
@@ -87,15 +87,17 @@ def checkSameMetadata(labeled_df, meta):
 
     return checkResult
 
-def getDistanceHeatmapDict(df):
+def getDistanceHeatmapDict(labeled_df, df):
     """
     Generate Distance Heatmap Dictitonary List for overview 
     by grouping the results and extracting distances from result table.
 
     Parameters
     -----------
+    labeled_df : DataFrame
+        LabeledDataFrame   
     df : DataFrame
-        LabeledDataFrame    
+        dataframe
```

Comment:
Update the docstring to describe that "The df parameter will be passed by filter_df, detect_df and rank_df, not always result_df."